src/main/cli/process.ts

```diff
@@ -0,0 +1,111 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import * as debug_ from "debug";
+import { app } from "electron";
+import * as path from "path";
+import { _PACKAGING } from "readium-desktop/preprocessor-directives";
+import * as yargs from "yargs";
+import { cli_, cliImport, cliOpds, cliRead } from "./commandLine";
+
+declare const __APP_NAME__: string;
+
+// Logger
+const debug = debug_("readium-desktop:cli");
+
+export function cli(mainFct: () => void) {
+
+    yargs
+        .scriptName(__APP_NAME__)
```

Comment:
This is great! :)

May I suggest we add `.version(__APP_VERSION__)` as well?

https://github.com/yargs/yargs/blob/master/docs/api.md#versionversionboolean

Reply:
I can do that
but 
`If no arguments are passed to version (.version()), yargs will parse the package.json of your module and use its version value.` is the same that __APP_VERSION

Reply:
`__APP_VERSION__` is a build-time "preprocessor directive" global.

Yargs can fetch `package.json` at runtime if it knows where it is, but its default location may not work in all contexts (ASAR, `__dirname` inside the app JS bundle, etc.).

So, better provide the version reliably, via the existing `__APP_VERSION__` build-time global.

Reply:
done